app/controllers/qa/issues_controller.rb

```diff
@@ -23,7 +23,8 @@ def edit
   def update
     if @issue.update(state: @state, updated_at: Time.now)
       track_state_change(@issue)
-      redirect_to project_qa_issues_path(current_project), notice: 'State updated successfully.'
+
+      redirect_to_next_issue_or_index
```

Comment:
from an esthetics point of view, I would have preferred something like `redirect_to next_issue_or_index_path` and have the helper return a path. More familiar.

I see now it may be because of the :notice param :( although potentially the method could return **[path, notice: ""] which could be fed to `redirect_to`... not sure if too fancy.